test/playback.test.js

```diff
@@ -105,7 +119,7 @@ QUnit.test('playlist with fmp4 segments', function(assert) {
   });
 
   player.src({
-    src: 'https://storage.googleapis.com/shaka-demo-assets/angel-one-hls/hls.m3u8',
+    src: 'https://devstreaming-cdn.apple.com/videos/streaming/examples/bipbop_adv_example_hevc/master.m3u8',
```

Comment:
why the source change?

Reply:
The apple source seems to load a lot quicker, but we can go back if we want. It was only added for TBA